playbooks/eucalyptus_basic_server.yml

```diff
@@ -0,0 +1,77 @@
+---
+# for single server community
+# This file based on `edx_sandbox.yml`
+
+- name: Configure instance(s)
+  hosts: all
+  become: True
```

Comment:
@mrgnr I've found this comment in the `sudo` role:
> # Workaround. What we really want to do is `become_method: sudo su` to run
> # tasks as other users. That's not currently possible in Ansible, but may be in
> # the future: https://github.com/ansible/ansible/issues/12686. Note that
> # modifying /etc/sudoers requires root privileges, so the user running this
> # task must already have the ability to sudo as root.

And also, I noticed that `amc.yml` is using this role in the playbook, maybe is the way to go, any thoughts?